contributors.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: page
+title: Contributors
+---
+
+## Active PIs
```

Comment:
Perhaps "Project leaders" would be better? Not everybody knows what a "PI" is, plus "active principal investigator" sounds odd to me.

Anyway, from my point of view, we could merge this and then iterate later. I still wonder if @micjoswig has any direct feedback, though? I.e. based on the preview at https://rfourquet.github.io/oscar-website/contributors/ resp. https://rfourquet.github.io/oscar-website/credits/

Reply:
> Not everybody knows what a "PI" 

Agreed, I think the majority of people wouldn't know, and even the un-abbreviated "principal investigator" would not be clear for everyone. I updated with "Project leaders" as you suggested.